### PR TITLE
(WIP) Switch over to Pushwoosh, and use request instead of nzero-push module

### PR DESCRIPTION
**Pull request by robbiecarlton**
This should not be merged until the new apps are in the app store.

### CODE REVIEW

File: api/models/PushNotification.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 var format = require('util').format
 var Promise = require('bluebird')
-var ZeroPush = require('nzero-push')
+var request = require('request')
 var rollbar = require('rollbar')
+var HTTPStatus = require('http-status')
```
Comment (levity) [inline api/models/PushNotification.js]: No change to package.json for this?

Comment (levity) [inline api/models/PushNotification.js]: Oh, WIP, never mind

**Comment by robbiecarlton** [comment]
@levity FYI, The test that's failing is also failing in the master branch. Don't know how that happened? It's in the Digest tests

**Comment by levity** [comment]
huh? the test i see failing is just related to http-status not being in package.json.

**Comment by robbiecarlton** [comment]
k, yup, my confusion. Locally I'm getting a test failing on the digest. I'll investigate.

**Pull request closed without merging**